tikv-client/src/main/java/com/pingcap/tikv/codec/CodecDataInput.java

```diff
@@ -187,9 +194,18 @@ public int readUnsignedShort() {
     }
   }
 
+  public int readUnsignedShortLSB() {
+    LittleEndianDataInputStream ldInputStream = new LittleEndianDataInputStream(inputStream);
+    try {
+      return ldInputStream.readUnsignedShort();
+    } catch (Exception e) {
+      throw new RuntimeException(e);
+    }
+  }
+
   public int readPartialUnsignedShort() {
```

Comment:
This is not used anymore? If so why don't just remove it.

Reply:
ok. i will remove it.